scene/gui/color_picker.cpp

```diff
@@ -745,7 +748,11 @@ void ColorPicker::_update_presets() {
 		String cached_name = editor_settings->call(SNAME("get_project_metadata"), "color_picker", "palette_name", String());
 		palette_path = editor_settings->call(SNAME("get_project_metadata"), "color_picker", "palette_path", String());
 		bool palette_edited = editor_settings->call(SNAME("get_project_metadata"), "color_picker", "palette_edited", false);
-		if (!cached_name.is_empty()) {
+		if (cached_name.is_empty()) {
+			palette_name->set_text("");
```

Comment:
Is there any use in removing the text if the label is getting hidden?

Reply:
Good point. I simply did it the same way it is done elsewhere - https://github.com/godotengine/godot/blob/240861f41962b6570594ec83ecae004e5793ae5f/scene/gui/color_picker.cpp#L1144

Should I remove it in both places?

Reply:
Yeah, you could.